test/compile/stack.test.ts

```diff
@@ -1,3 +1,21 @@
 /* tslint:disable:quotemark */
 
-// TODO: write properly
+import {assert} from 'chai';
+import {parseUnitModel} from '../util';
+
+describe('Stack', function() {
+  describe('compileStackProperties()', function() {
+    it('should return undefined from ranged bars with aggregate', function() {
```

Comment:
For test coverage, you should test a case with `y` and `y2` too.  
(Take 30 seconds more to do, so doesn't hurt to do it.)